public/app/scripts/Authentication/user.js

```diff
@@ -12,7 +12,7 @@
         function init() {
             return getRecipes().then(function(recipes) {
                 context.recipes = recipes;
-                context.reloadRecipes = getRecipes;
+                context.reloadRecipes = true;
```

Comment:
I use this basically as a dirty flag to mark if the database needs to be hit or not upon a data request.